Makefile

```diff
@@ -23,6 +23,17 @@ go:
 	@echo "Generating go protobuf files"
 	docker run --pull always --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -d protos -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
 
+python: env/pyenv.cfg
+	@echo "Generating python protobuf files"
+	. env/bin/activate && cd ./gen/pb-python/sigstore_protobuf_specs && python -m grpc_tools.protoc -I ../../../protos/ --python_betterproto_out=. ../../../protos/*
+
+env/pyenv.cfg:
```

Comment:
Though venvs are definitely a *better* solution for Python than Docker, they're a less consistent one. Ideally, you would be able to clone this repo on a machine without Python installed and still update it with a simple `make all`. Is is possible to move these steps *inside* of a Docker container? (You can even keep them exposed as Make targets to run them directly if you'd like)